src/main/java/lv/ctco/javaschool/app/boundary/TripApi.java

```diff
@@ -48,4 +66,60 @@ public ListTripDto getActiveTrips() {
                 .collect(Collectors.toList()));
         return listTripDto;
     }
+
+    @GET
+    @Path("/places")
+    @Produces("application/json")
+    @RolesAllowed({"ADMIN", "USER"})
+    public List<Place> getAllPlaces() {
+        return Arrays.asList(Place.values());
+    }
+
+    @POST
+    @RolesAllowed({"ADMIN", "USER"})
+    @Path("/createTrip")
+    public void createNewTrip(JsonObject field) {
```

Comment:
Maybe it's better to use a specific DTO object instead of manipulating with low-level JsonObject?